lib/sizzle.patched.js

```diff
@@ -1488,6 +1491,20 @@ Expr = Sizzle.selectors = {
 
 		"contains": markFunction(function( text ) {
 			text = text.replace( runescape, funescape );
+
+			// Patch #5: Support regular expressions as arguments of :contains pseudo class.
+			if(/^\s*\/.*\/\s*$/.test(text)) {
+				try {
+					var regex = new RegExp(text.trim().slice(1, -1));
```

Comment:
Frankly, I suppose that it's wrong to have it defined here and then used outside of the `try {}` scope. Javascript supports it, but I find it unclear, let's move the definition above.